targettests/execution/sudoku_2x2-1.cpp

```diff
@@ -9,6 +9,7 @@
 // REQUIRES: c++20
 // clang-format off
 // RUN: nvq++ --target anyon                    --emulate %s -o %t && %t | FileCheck %s
+// RUN: nvq++ --target braket                   --emulate %s -o %t && %t | FileCheck %s
```

Comment:
As per issue# https://github.com/NVIDIA/cuda-quantum/issues/2325, this isn't expected to pass (yet), right?